core/tests/puppeteer-acceptance-tests/user-utilities/logged-in-users-utils.ts

```diff
@@ -152,6 +178,34 @@ export class LoggedInUser extends BaseUser {
     await this.goto(getStartedUrl);
   }
 
+  /**
+   * Function to navigate to the Creator Guidelines page.
+   */
+  async clickOnCreatorGuidelinesLinkinFooter(): Promise<void> {
```

Comment:
This function should click the button "Creator Guidelines" in the Teach/Learn Footer, not navigate to the Creator Guidelines Link with the goto function.

Reply:
Yeah I renamed some symbols incorrectly on accident. Should be fixed now